config/network-policies.yaml

```diff
@@ -0,0 +1,563 @@
+          name: eirini
+      namespaceSelector:
+        matchLabels:
+          cf-for-k8s.cloudfoundry.org/cf-system-ns: ""
+    - podSelector:
+        matchLabels:
+          app: log-cache
+      namespaceSelector:
+        matchLabels:
+          cf-for-k8s.cloudfoundry.org/cf-system-ns: ""
+  egress:
+  - {}
+---
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+metadata:
+  name: cf-api-deployment-updater
+  namespace: #@ data.values.system_namespace
+spec:
+  policyTypes:
+  - Egress
+  podSelector:
+    matchLabels:
+      app.kubernetes.io/name: cf-api-deployment-updater
+  egress:
+  - to:
+    - namespaceSelector:
+        matchLabels:
+          name: cf-db
+      podSelector:
+        matchLabels:
+          release: cf-db
+    - namespaceSelector:
+        matchLabels:
+          cf-for-k8s.cloudfoundry.org/cf-system-ns: ""
+      podSelector:
+        matchLabels:
+          name: eirini
+---
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+metadata:
+  name: cf-api-clock
+  namespace: #@ data.values.system_namespace
+spec:
+  policyTypes:
+  - Egress
+  podSelector:
+    matchLabels:
+      app.kubernetes.io/name: cf-api-clock
+  egress:
+  - to:
+    - namespaceSelector:
+        matchLabels:
+          name: cf-db
+      podSelector:
+        matchLabels:
+          release: cf-db
+  - to:
+    - namespaceSelector:
+        matchLabels:
+          cf-for-k8s.cloudfoundry.org/cf-system-ns: ""
+      podSelector:
+        matchLabels:
+          name: eirini
+---
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+metadata:
+  name: cf-api-worker
+  namespace: #@ data.values.system_namespace
+spec:
+  policyTypes:
+  - Egress
+  podSelector:
+    matchLabels:
+      app.kubernetes.io/name: cf-api-worker
+  egress:
+  - {}
+---
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+metadata:
+  name: ccdb-migrate
+  namespace: #@ data.values.system_namespace
+spec:
+  policyTypes:
+  - Egress
+  podSelector:
+    matchLabels:
+      job-name: ccdb-migrate
+  egress:
+  - to:
+    - namespaceSelector:
+        matchLabels:
+          name: cf-db
+      podSelector:
+        matchLabels:
+          release: cf-db
+---
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+metadata:
+  name: cf-blobstore-minio
+  namespace: cf-blobstore
+spec:
+  policyTypes:
+  - Ingress
+  podSelector:
+    matchLabels:
+      release: cf-blobstore
+  ingress:
+  - from:
+    - podSelector:
+        matchLabels:
+          istio: ingressgateway
+      namespaceSelector:
+        matchLabels:
+          cf-for-k8s.cloudfoundry.org/istio-system-ns: ""
+    - podSelector:
+        matchLabels:
+          cloudfoundry.org/source_type: STG
+      namespaceSelector:
+        matchLabels:
+          name: #@ data.values.staging_namespace
+    - namespaceSelector:
+        matchLabels:
+          cf-for-k8s.cloudfoundry.org/cf-system-ns: ""
+      podSelector:
+        matchLabels:
+          app.kubernetes.io/name: cf-api-server
```

Comment:
We might need to ad the cf-api-worker to this since it may be used to delete from the blobstore.

Reply:
@piyalibanerjee made this change:
https://github.com/cloudfoundry/cf-for-k8s/pull/270/commits/1b0f40ef1ba6f7b7fc0b4b4a3d21ed8698930b9b

Added policy for any Cloud Controller component that has a `fog_connection` config.